lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -561,6 +561,60 @@ ospf:
     config_get_token: ['/^router ospf <name>$/i', '/^vrf\s+(\S+)$/']
     config_set: ["router ospf <name>", "<state> vrf <vrf>", "end"]
 
+radius_server:
+  hosts:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/'
+    config_set: '<state> radius-server host <ip>'
+    default_value: ''
+
+  auth-port:
+    config_get: "show running-config radius"
+    config_get_token: '/^radius-server host %s.* auth-port (\d+).*$/'
+    config_set: '<state> radius-server host <ip> auth-port <port>'
+    default_value: ''
```

Comment:
2,3 and 4 are great, thanks! I'll add this to the code.

Regarding timeout value, putting timeout 5 in the CLI results in this setting being displayed. As such, from puppet's perspective, the default value is not 5, but 'not there'. Any ideas?